diff --git a/include/openssl/crypto.h b/include/CBigNumBoringSSL_crypto.h
index a22bfa2..865c1cb 100644
--- a/include/openssl/crypto.h
+++ b/include/CBigNumBoringSSL_crypto.h
@@ -15,16 +15,16 @@
 #ifndef OPENSSL_HEADER_CRYPTO_H
 #define OPENSSL_HEADER_CRYPTO_H
 
-#include <openssl/base.h>  // IWYU pragma: export
-#include <openssl/sha.h>
+#include "CBigNumBoringSSL_base.h"  // IWYU pragma: export
+#include "CBigNumBoringSSL_sha.h"
 
 // Upstream OpenSSL defines |OPENSSL_malloc|, etc., in crypto.h rather than
 // mem.h.
-#include <openssl/mem.h>
+#include "CBigNumBoringSSL_mem.h"
 
 // Upstream OpenSSL defines |CRYPTO_LOCK|, etc., in crypto.h rather than
 // thread.h.
-#include <openssl/thread.h>
+#include "CBigNumBoringSSL_thread.h"
 
 
 #if defined(__cplusplus)
