diff --git a/include/openssl/ctrdrbg.h b/include/CBigNumBoringSSL_ctrdrbg.h
index 849fc82..dc4d87c 100644
--- a/include/openssl/ctrdrbg.h
+++ b/include/CBigNumBoringSSL_ctrdrbg.h
@@ -15,7 +15,7 @@
 #ifndef OPENSSL_HEADER_CTRDRBG_H
 #define OPENSSL_HEADER_CTRDRBG_H
 
-#include <openssl/base.h>  // IWYU pragma: export
+#include "CBigNumBoringSSL_base.h"  // IWYU pragma: export
 
 #if defined(__cplusplus)
 extern "C" {
