diff --git a/include/openssl/prefix_symbols_internal_c.h b/include/CBigNumBoringSSL_prefix_symbols_internal_c.h
index 6e84071..244761e 100644
--- a/include/openssl/prefix_symbols_internal_c.h
+++ b/include/CBigNumBoringSSL_prefix_symbols_internal_c.h
@@ -19,7 +19,7 @@
 #ifndef OPENSSL_HEADER_PREFIX_SYMBOLS_INTERNAL_C_H
 #define OPENSSL_HEADER_PREFIX_SYMBOLS_INTERNAL_C_H
 
-#include <openssl/prefix_symbols.h>
+#include "CBigNumBoringSSL_prefix_symbols.h"
 
 
 #if defined(BORINGSSL_PREFIX)
