diff --git a/gen/bcm/sha256-586-linux.S b/gen/bcm/sha256-586-linux.S
index 41b3759..b7fa26f 100644
--- a/gen/bcm/sha256-586-linux.S
+++ b/gen/bcm/sha256-586-linux.S
@@ -1,7 +1,8 @@
+#define BORINGSSL_PREFIX CBigNumBoringSSL
 // This file is generated from a similarly-named Perl script in the BoringSSL
 // source tree. Do not edit by hand.
 
-#include <openssl/asm_base.h>
+#include <CBigNumBoringSSL_asm_base.h>
 
 #if !defined(OPENSSL_NO_ASM) && defined(OPENSSL_X86) && defined(__ELF__)
 .text
