diff --git a/Sources/CBigNumBoringSSL/include/CBigNumBoringSSL_bn.h b/Sources/CBigNumBoringSSL/include/CBigNumBoringSSL_bn.h
--- a/Sources/CBigNumBoringSSL/include/CBigNumBoringSSL_bn.h
+++ b/Sources/CBigNumBoringSSL/include/CBigNumBoringSSL_bn.h
@@ -18,7 +18,8 @@

 #include "CBigNumBoringSSL_base.h"   // IWYU pragma: export

-#include <inttypes.h>  // for PRIu64 and friends
+// Use inttypes.h for PRIu64, but only in non-Swift contexts
+#include <inttypes.h>
 #include <stdio.h>  // for FILE*

 #if defined(__cplusplus)
