diff --git a/Sources/CBigNumBoringSSL/crypto/hrss/hrss.cc b/Sources/CBigNumBoringSSL/crypto/hrss/hrss.cc
index 93a214e..eee4e58 100644
--- a/Sources/CBigNumBoringSSL/crypto/hrss/hrss.cc
+++ b/Sources/CBigNumBoringSSL/crypto/hrss/hrss.cc
@@ -13,6 +13,7 @@
  * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE. */

 #include <CBigNumBoringSSL_hrss.h>
+#include <inttypes.h>

 #include <assert.h>
 #include <stdio.h>
diff --git a/Sources/CBigNumBoringSSL/include/CBigNumBoringSSL_bn.h b/Sources/CBigNumBoringSSL/include/CBigNumBoringSSL_bn.h
index c86c1ef..7013140 100644
--- a/Sources/CBigNumBoringSSL/include/CBigNumBoringSSL_bn.h
+++ b/Sources/CBigNumBoringSSL/include/CBigNumBoringSSL_bn.h
@@ -126,7 +126,8 @@
 #include "CBigNumBoringSSL_base.h"
 #include "CBigNumBoringSSL_thread.h"

 #include <inttypes.h>  // for PRIu64 and friends
+#include <sys/types.h>
 #include <stdio.h>  // for FILE*

 #if defined(__cplusplus)
