diff --git a/crypto/fipsmodule/bn/bn.cc.inc b/crypto/fipsmodule/bn/bn.cc.inc
index 38dea25..e4c9794 100644
--- a/crypto/fipsmodule/bn/bn.cc.inc
+++ b/crypto/fipsmodule/bn/bn.cc.inc
@@ -12,14 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include <openssl/bn.h>
+#include <CBigNumBoringSSL_bn.h>
 
 #include <assert.h>
 #include <limits.h>
 #include <string.h>
 
-#include <openssl/err.h>
-#include <openssl/mem.h>
+#include <CBigNumBoringSSL_err.h>
+#include <CBigNumBoringSSL_mem.h>
 
 #include "../../mem_internal.h"
 #include "../delocate.h"
