diff --git a/crypto/fipsmodule/bn/gcd.cc.inc b/crypto/fipsmodule/bn/gcd.cc.inc
index 4fdb2e9..cda22f4 100644
--- a/crypto/fipsmodule/bn/gcd.cc.inc
+++ b/crypto/fipsmodule/bn/gcd.cc.inc
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include <openssl/bn.h>
+#include <CBigNumBoringSSL_bn.h>
 
-#include <openssl/err.h>
+#include <CBigNumBoringSSL_err.h>
 
 #include "internal.h"
 
